src/Orderable.js

```diff
@@ -0,0 +1,31 @@
+module.exports = (Class) => class extends Class {
+  before(name) {
+    if (this.__after) {
+      throw new Error(`Unable to set .before(${JSON.stringify(name)}) with existing value for after`);
```

Comment:
Does the `with existing value for after` part need quotes or something around the "after", to make it clearer? (Or else `.after()` ?)

(And same for below)